src/index.js

```diff
@@ -222,7 +231,7 @@ export async function run() {
       if (tmateWeb) {
         core.info(`Web shell: ${tmateWeb}`);
       }
-      core.info(`SSH: ${tmateSSH}`);
+      core.info(`SSH: ssh -p ${port} ${token}@${host}`);
```

Comment:
the default `tmateSSH` information is incorrect. It is unable to pick up correct IP address and port from input.